_infra/helm/case/templates/deployment.yaml

```diff
@@ -127,10 +127,6 @@ spec:
             value: /var/secrets/google/credentials.json
           - name: GOOGLE_CLOUD_PROJECT
             value: "{{ .Values.gcp.project }}"
-          - name: GCP_PROJECT
```

Comment:
If you delete this then it needs to be replaced with something.  Currently the project field is populated by 
```
gcp:
  project:
```
I know theres a google_cloud_project field but it's not being used by the pubsub stuff

Reply:
no this was a duplicate. you created this key down in the deployment.yaml I missed to see it.

Reply:
so I did.  Sorry about that ^^